ds701_book/jupyter_notebooks/15-Classification-II-kNN.ipynb

```diff
@@ -36,19 +36,23 @@
       "cell_type": "markdown",
```

Comment:
<p>hmm... strange. this seems to be from my earlier PR that is already merged. Why is it showing as a difference here? Perhaps if you didn't make changes here then you can remove these 2 other notebooks from this PR.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tools4ds/DS701-Course-Notes/pull/145/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Same thing here. The PR diff shows just a minor change to the plot text whereas NBReview is showing still all the changes I originally made. We can disregard this too.